app/serializers/v1/subscription_serializer.rb

```diff
@@ -12,7 +12,7 @@ def serialize
         plan_code: model.plan.code,
         status: model.status,
         billing_time: model.billing_time,
-        subscription_date: model.subscription_date&.iso8601,
+        subscription_date: model.subscription_at&.to_date&.iso8601,
```

Comment:
we are not exposing `subscription_at` here?

Reply:
It will be, but in a next PR, this one is about changing the datatype internaly without breaking the current API/GraphQL